app/src/main/java/com/capstone/weatherapp/WeatherCache.kt

```diff
@@ -0,0 +1,72 @@
+package com.capstone.weatherapp
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class CityCache(
+    @PrimaryKey val id: Int,
+    @ColumnInfo(name = "city") val name: String,
```

Comment:
You don't need this annotation here if you don't want, it's just used to rename the attributes in the table. if you remove it, this field will just be called "name" in the DB table. up to you